pandas/tests/io/json/test_normalize.py

```diff
@@ -226,6 +237,21 @@ def test_non_ascii_key(self):
         result = json_normalize(json.loads(testjson))
         tm.assert_frame_equal(result, expected)
 
+    def test_missing_field(self, author_missing_data):
+        result = json_normalize(author_missing_data)
```

Comment:
can you add the issue number as a comment. & a 1-liner explaining the test purpose